app/assets/stylesheets/utilities.scss

```diff
@@ -1,6 +1,10 @@
 .push-down {
   margin-bottom: 1rem;
 }
+
+.push-down-2x {
+  margin-bottom: 2rem;
+}
```

Comment:
Can we have a newline between this and the push-down-3x rule please? just to keep it consistent with the other rules.

Reply:
Of course, I’ll add that later this afternoon.